### PR TITLE
Refine storage utilization page

### PR DESCRIPTION
**Pull request by archichen**
Change all tables to thumbnail. The larger table is shown when the user clicks on the thumbnail.

Thumbnails overview
thumbnails

Large table
lt1

Large table
lt2

### CODE REVIEW

**Comment by philo-he** [comment]
image

**Comment by philo-he** [comment]
1)In large figure, the usage value should not be shaded as the above picture shows. 
2)Moving small figures to central position looks better.

**Comment by archichen** [comment]
> 1)In large figure, the usage value should not be shaded as the above picture shows.
> 2)Moving small figures to central position looks better.

I have fixed first problem. But the second problem, small figure may painted by NVD3. So I cant change the position of it without restructure NVD3 source code.

**Comment by archichen** [comment]
@PHILO-HE 
Screenshot from 2019-03-11 16-25-02

**Comment by philo-he** [comment]
image

The java code style is not desirable. Set the code style as the picture shows in File/Settings/Editors/Code Style/Java and reformat the selecting code by clicking Code/Reformat the code.

File: smart-common/src/main/java/org/smartdata/NodeHosts.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+package org.smartdata;
```
Comment (philo-he) [inline smart-common/src/main/java/org/smartdata/NodeHosts.java:18]: Add license declaration as same as other src files.

File: smart-zeppelin/zeppelin-server/src/main/java/org/smartdata/server/rest/SystemRestApi.java [hunk 2]
```diff
@@ -60,4 +60,16 @@ public Response agents() {
...
+
+  @GET
+  @Path("/all_agent_hosts")
+  public Response allAgentHosts() {
+    return new JsonResponse<>(Response.Status.OK, smartEngine.getAgentHosts()).build();
+  }
+
+  @GET
+  @Path("/all_server_hosts")
+  public Response allMasterHosts() {
```
Comment (philo-he) [inline smart-zeppelin/zeppelin-server/src/main/java/org/smartdata/server/rest/SystemRestApi.java:72]: allServerHosts

**Comment by philo-he** [comment]
src/index.html
[INFO]       link rel="stylesheet" href="app/dashboard/views/cluster/nodeinfo/nodeStatus.css"
[INFO]                ^ The empty tag : [ link ] must be self closed. (tag-self-close)
[INFO] 
[INFO] >> 1 errors in 1 files
[INFO] Warning: Task "htmlhint:src" failed. Use --force to continue.
[INFO] 
[INFO] Aborted due to warnings.

**Comment by archichen** [comment]
@PHILO-HE I have fixed bug that open index page blank after compiled code. Also I have compiled source code and tested at local, work well.

**Pull request merged**